GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+# Kiali Governance
+
+This document defines governance policies for the Kiali project.
+
+## Maintainers
+
+Kiali Maintainers have write access to the Kiali GitHub repositories https://github.com/kiali.
+They can merge their own patches or patches from others. The current maintainers can be found in https://github.com/orgs/kiali/teams/maintainers.
+
+This privilege is granted with some expectation of responsibility: maintainers are people who care about the Kiali project and want to help it grow and improve. A maintainer is not just someone who can make changes, but someone who has demonstrated his or her ability to collaborate with the team, get the most knowledgeable people to review code, contribute high-quality code, and follow through to fix issues (in code or tests).
+
+A maintainer is a contributor to the Kiali project's success and a citizen helping the project succeed.
+
+## Becoming a Maintainer
+
+To become a maintainer you need to demonstrate the following:
+
+  * commitment to the project
+    * participate in discussions, contributions, code reviews for 3 months or more,
+    * perform code reviews for 10 non-trivial pull requests,
+    * contribute 10 non-trivial pull requests and have them merged into master,
```

Comment:
I think the term "master" should no longer be used. Probably "main"?

Reply:
Without enter in discussions, for the case of Kiali, the project continues using the "master" branch.
So, I'd add that expression "master branch" to avoid any potential context misunderstanding, but let's not use a different rename of the branch that may cause confusion.

Reply:
I thought that github is renaming master to main for projects, but I guess I misunderstood

Reply:
The status is that for new repos, github create a "main" branch instead of "master" branch.
For existing projects is untouched.

As a potential migration may have more implications, my suggestion is to use the "master branch" wording just for this doc and with that context.

If in the future we trigger a migration (we have several repos and it's not a transparent process) we can update the doc as well (just to refer to the same branch we use).

Reply:
if the branch keeps being named master then the governance should certainly match it. as I said, my understanding around the renaming was not accurate.